### PR TITLE
Automatic password validation + isPhoneNumber() assertion #17

### PR DESCRIPTION
**Pull request by SuuSoJeat**
Resolved issue #17

### CODE REVIEW

File: library/src/main/java/com/afollestad/vvalidator/assertion/input/InputAssertions.kt [hunk 1]
```diff
@@ -318,6 +327,25 @@ sealed class InputAssertions {
...
+    private var ignoreCase: Boolean = false
+
+    /** Case is ignored when comparing the two input text. */
+    fun ignoreCase(): ComparesAssertion {
+      ignoreCase = true
+      return this
+    }
+
+    override fun isValid(view: EditText): Boolean {
+      return view.text().compareTo(targetView.text(), ignoreCase = ignoreCase) == 0
```
Comment (afollestad) [inline library/src/main/java/com/afollestad/vvalidator/assertion/input/InputAssertions.kt]: This is the same as `equalsIgnoreCase(...)` 🙂

Comment (SuuSoJeat) [reply library/src/main/java/com/afollestad/vvalidator/assertion/input/InputAssertions.kt]: I'm sorry. I don't quite get what you meant.

Comment (SuuSoJeat) [reply library/src/main/java/com/afollestad/vvalidator/assertion/input/InputAssertions.kt]: Sorry, i was busy with exams in the past week.

**Comment by SuuSoJeat** [comment]
I can't fix the unit test failing on assertion description res.

**Pull request closed without merging**